Dockerfile

```diff
@@ -72,11 +73,12 @@ RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && locale-gen
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
+ENV VAULT_ENC_KEY aHD8DZRdk2emnkdktFZRh3E9RNg4aOY7
```

Comment:
This variable should be defined in the docker-compose file, as it's a secret for passwords. Here's more info about the enc lib https://hexdocs.pm/cloak_ecto/readme.html